composer.json

```diff
@@ -2,12 +2,12 @@
     "name": "upscale/doctrine-value-object-bundle",
     "description": "Symfony bundle for yokai/doctrine-value-object",
     "type": "symfony-bundle",
-    "version": "1.0.1",
+    "version": "1.0.2",
     "license": "Apache-2.0",
     "require": {
         "php": "^8.1",
-        "symfony/framework-bundle": "^6.1",
-        "yokai/doctrine-value-object": "^0.0.3"
+        "symfony/framework-bundle": "^6.1|^7.0",
+        "yokai/doctrine-value-object": "^0.0.3|^0.2"
```

Comment:
Could you please include the `^0.1` version as well ti not have the version gap?